common/services/prismic/events.js

```diff
@@ -61,14 +61,19 @@ function parseEventDoc(document: PrismicDocument, scheduleDocs?: PrismicFragment
   const eventbriteIdScheme = identifiers.find(id => id.identifierScheme === 'eventbrite-id');
   const eventbriteId = eventbriteIdScheme && eventbriteIdScheme.value;
 
+  const audiences = document.data.audiences.map(audience => isDocumentLink(audience.audience) ? ({
+    title: asText(audience.audience.data.title),
+    description: asText(audience.audience.data.description)
```

Comment:
We don't use the description anywhere, but if we did, it should probably just be in prismic format.

Reply:
Yep, have changed that here: https://github.com/wellcometrust/wellcomecollection.org/pull/2921/commits/1b84351be0a603b943e9c796eb4365eee6e9e2cc#diff-db1ea758ff420e1c26768041df9cadf8R74

Reply:
👍